php/src/Google/Protobuf/Internal/GPBUtil.php

```diff
@@ -252,9 +252,23 @@ public static function getClassNamePrefix(
             return $prefix;
         }
 
-        $reserved_words = array("Empty", "ECHO", "ARRAY");
+        $reserved_words = array(
```

Comment:
Perhaps this could be an associative array, and then the check could be done with a lookup instead of a loop?

Reply:
Done